src/components/transport_manager/include/transport_manager/websocket/websocket_connection.h

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
To help differentiate from the cloud websocket transport directory, can this websocketdirectory be renamed to `websocket_server`?

Reply:
@JackLivio Sure. Agree. I will inform you with the commit hash.

Reply:
@JackLivio Fixed in 41f89d9e65d710d16cd2ec8160a85a5a7e833cdb